src/python/tools/Report_InvalidUrls_FindHttpURLs.csv

```diff
@@ -0,0 +1,18 @@
+url	relativeFilepath
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\CommonCpp.props
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\NativeBridge\NativeBridge.vcxproj
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\python\nimbusml.pyproj
+http://www.nvidia.com/Drivers	.\src\DotNetBridge\MessageValidator.cs
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\docs\docstrings\FastLinearBinaryClassifier.txt
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\docs\docstrings\FastLinearClassifier.txt
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\docs\docstrings\FastLinearRegressor.txt
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\internal\core\linear_model\fastlinearbinaryclassifier.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\internal\core\linear_model\fastlinearclassifier.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\internal\core\linear_model\fastlinearregressor.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\linear_model\fastlinearbinaryclassifier.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\linear_model\fastlinearclassifier.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\linear_model\fastlinearregressor.py
+http://users.cms.caltech.edu/~jtropp/papers/HMT11-Finding-Structure-	.\src\python\docs\docstrings\PcaAnomalyDetector.txt
+http://users.cms.caltech.edu/~jtropp/papers/HMT11-Finding-Structure-	.\src\python\nimbusml\decomposition\pcaanomalydetector.py
+http://users.cms.caltech.edu/~jtropp/papers/HMT11-Finding-Structure-	.\src\python\nimbusml\internal\core\decomposition\pcaanomalydetector.py
+http://www2.spsc.tugraz.at/www-	.\src\python\docs\docstrings\SigmoidKernel.txt
```

Comment:
>http://www2.spsc.tugraz.at/www [](start = 0, length = 30)

url in source file is different

Reply:
Indeed, the correct link is `https://www2.spsc.tugraz.at//www-archive/AdvancedSignalProcessing/WS05-Mistral/advances.pdf`. This website does support HTTPS. This link was also cut into two lines, hence why FindHttpURLs.py didn't pick it up as a non-alterable HTTP URL.

Reply:
i know the reason, are you going to fix it?

---
In reply to: [337778958](https://github.com/microsoft/NimbusML/pull/346#discussion_r337778958) [](ancestors = 337778958)

Reply:
I believe I fixed it in commit [75a6a53](https://github.com/microsoft/NimbusML/pull/346/commits/75a6a536cb2283191823422384c7b6f99efa3931)

Reply:
yes, but why this url is still in the report then? If its valid url it should not be here. 

---
In reply to: [337788743](https://github.com/microsoft/NimbusML/pull/346#discussion_r337788743) [](ancestors = 337788743,337778958)

Reply:
It is still there because the URL in the file is broken into two lines, with a "\n" in between the two lines, in SigmoidKernel.txt. As such it is not a valid URL in a strict sense. It will disappear if I remove the line in the middle, should I do that?

Reply:
I am ok with that. Pls do same for other split urls too

---
In reply to: [337813735](https://github.com/microsoft/NimbusML/pull/346#discussion_r337813735) [](ancestors = 337813735)